src/views/InvolvementPanel/FormViews/TeacherFormView.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import FormView from './FormView';
+
+export default class TeacherFormView extends React.Component {
+ render() {
+
+  const formMessage = "Join our teaching team and change a child’s life. We have a wide range of teaching opportunities available, from going into classrooms in cities across the country to teaching from the comfort of your own home.";
+  const inputFields = [
```

Comment:
Can you reach out to Lisa/Kiera and confirm that these are the fields we want on this form?